GameData/RealismOverhaul/RO_SuggestedMods/MK2_Expansion/MK2_FuelTank.cfg

```diff
@@ -0,0 +1,208 @@
+{
+    !RESOURCE[LiquidFuel] {}
+    !RESOURCE[Oxidizer] {}
+    !RESOURCE[MonoPropellant] {}
+}
+
+@PART[M2X_Shortbicoupler|M2X_HypersonicNose|M2X_InverterFuselage|M2X_linearTricoupler|M2X_Mk2bicoupler|M2X_UST|M2X_Short25adapter|M2X_SlantAdapterS|M2X_SpadeTail|M2X_625tricoupler]:FOR[RealismOverhaul]
+{
+	%skinMaxTemp = 2500
+}
+
+@PART[M2X_Shortbicoupler]:FOR[RealismOverhaul]
+{
+    %RSSROConfig = True
+    %rescaleFactor = 1.722222
+    @mass = 0.4005    
+}
+
+@PART[M2X_HypersonicNose]:FOR[RealismOverhaul]
+{
+    %RSSROConfig = True
+    %rescaleFactor = 1.722222
+    @mass = 1.522
+
+    MODULE
+    {
+        name = ModuleFuelTanks
+        volume = 16000
+        type = Fuselage
+    }
+    
+    @MODULE[ModuleRCSFX*]
+    {
+        !resourceName = DELETE
+        thrusterPower = 2
+        PROPELLANT
+        {
+            name = MMH
+            ratio = 0.50
+        }
+        PROPELLANT
+        {
+            name = NTO
+            ratio = 0.50
+        }
+        atmosphereCurve
+        {
+            key = 0 280.84
+			key = 1 253.498
+        }
+
+    }
+}
+
+@PART[M2X_InverterFuselage]:FOR[RealismOverhaul]
+{
+    %RSSROConfig = True
+    %rescaleFactor = 1.722222
+    @mass = 1.522
+
+    MODULE
+    {
+        name = ModuleFuelTanks
+        volume = 32000
+        type = Fuselage
+    }
+
+}
+
+@PART[M2X_linearTricoupler]
+{
+    %RSSROConfig = True
+    %rescaleFactor = 1.722222
+    @mass = 1.469
+
+    MODULE
+    {
+        name = ModuleFuelTanks
+        volume = 32000
+        type = Fuselage
+    }
+}
+
+@PART[M2X_Mk2bicoupler]:FOR[RealismOverhaul]
+{
+    %RSSROConfig = True
+    %rescaleFactor = 1.722222
+    @mass = 0.881
+
+    MODULE
+    {
+        name = ModuleFuelTanks
+        volume = 32000
+        type = Fuselage
+    }
+}
+
+@PART[M2X_SupersonicNose]
+{
+    %RSSROConfig = True
+    %rescaleFactor = 1.722222
+    @mass = 0.4539
+
+    @MODULE[ModuleRCSFX*]
+    {
+        !resourceName = DELETE
+        thrusterPower = 2
+        PROPELLANT
+        {
+            name = MMH
+            ratio = 0.50
+        }
+        PROPELLANT
+        {
+            name = NTO
+            ratio = 0.50
+        }
+        atmosphereCurve
+        {
+            key = 0 280.84
+			key = 1 253.498
+        }
+    }
+}
+
+@PART[M2X_UST]:FOR[RealismOverhaul]
+{
+    %RSSROConfig = True
+    %rescaleFactor = 1.722222
+    @mass = 0.405
+
+    @MODULE[ModuleB9PartSwitch]
+    {
+        !SUBTYPE[#LOC_M2X_ServiceBay_STR|#LOC_M2X_ServiceBay_LF|#LOC_M2X_ServiceBay_LF/O|#LOC_M2X_ServiceBay_MONO|#LOC_M2X_ServiceBay_XE|#LOC_M2X_ServiceBay_MPXE] {}
+    }
+    
+    MODULE
+    {
+        name = ModuleFuelTanks
+        volume = 8000
+        type = Fuselage
+    }
+}
+
+@PART[M2X_Short25adapter]:FOR[RealismOverhaul]
+{
+    %RSSROConfig = True
+    %rescaleFactor = 1.722222
+    @mass = 0.774
```

Comment:
Seems too light, even lighter than the standard short fuselage despite being larger